include/villas/nodes/c37_118.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file
+ * @author Philipp Jungkamp <philipp.jungkamp@rwth-aachen.de>
+ * @copyright 2014-2022, Institute for Automation of Complex Power Systems, EONERC
+ * @license Apache 2.0
+ *********************************************************************************/
```

Comment:
Please check the new file headers. I think still style is not up-to-date anymore.